numpy/_core/tests/test_umath.py

```diff
@@ -694,11 +693,11 @@ def test_floor_division_corner_cases(self, dtype):
         with suppress_warnings() as sup:
             sup.filter(RuntimeWarning, "invalid value encountered in floor_divide")
             div = np.floor_divide(fnan, fone)
-            assert(np.isnan(div)), "dt: %s, div: %s" % (dt, div)
+            assert(np.isnan(div)), "div: %s" % div
             div = np.floor_divide(fone, fnan)
-            assert(np.isnan(div)), "dt: %s, div: %s" % (dt, div)
+            assert(np.isnan(div)), "div: %s" % div
             div = np.floor_divide(fnan, fzer)
-            assert(np.isnan(div)), "dt: %s, div: %s" % (dt, div)
+            assert(np.isnan(div)), "div: %s" % div
```

Comment:
Is there a reason to not remove the parentheses from the asserts?
```suggestion
            assert np.isnan(div), "div: %s" % div
```

Reply:
These commits contain automated fixes hen possible. I try not to manually modify these fixes, unless really needed. There's a ruff/pyupgrade rule for extraneous parentheses ([`UP034`](https://docs.astral.sh/ruff/rules/extraneous-parentheses/)), which I could apply in a different PR.